cloud-aws/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/AwsStackRequestHelper.java

```diff
@@ -255,6 +262,16 @@ private String getRootDeviceName(AuthenticatedContext ac, CloudStack cloudStack)
         return parameters;
     }
 
+    private String getDbFamily(AwsRdsInstanceView awsRdsInstanceView, AwsRdsDbParameterGroupView awsRdsDbParameterGroupView, boolean useSslEnforcement) {
+        if (!useSslEnforcement) {
+            return null;
+        }
+        if (!POSTGRES.equals(awsRdsInstanceView.getEngine())) {
+            throw new IllegalStateException("Unsupported RDS engine " + awsRdsInstanceView.getEngine());
+        }
+        return awsRdsInstanceView.getEngineVersion() == null ? POSTGRES + rdsMajorVersion : awsRdsDbParameterGroupView.getDBParameterGroupFamily();
```

Comment:
Why the need for this logic? I would just delegate to `awsRdsDbParameterGroupView.getDBParameterGroupFamily()` as the original expression in line 242 did.

Reply:
I can delegate.  Gergo asked if we can get the postgres (major) version from application yml, so now 
1) rds version (major.minor) can be in the rds request (in case of aws)
2) rds major version comes from application.yml if the request does not have version